processor.go

```diff
@@ -47,8 +48,34 @@ func (p *Processor) Members(orgName string) ([]string, error) {
 	return returnMembers, nil
 }
 
+func check(err error) {
+	if err != nil {
+		log.Fatalf("Something went wrong. Error message - %q", err)
+	}
+}
+
+func (p *Processor) getTeamsPaginated(orgName string) []*github.Team {
+	nextPage := 0
+	var allTeams []*github.Team
+	for {
+		opt := github.ListOptions{Page: nextPage, PerPage: 30}
+		teams, response, err := p.TeamsService.ListTeams(p.Context, orgName, &opt)
+		check(err)
```

Comment:
I used this pattern in the past a lot but decided to go away from it in favor of passing error to caller.
Then caller can decide how to deal with the error

```suggestion
func (p *Processor) getTeamsPaginated(orgName string) ([]*github.Team, error) {
	var (
		nextPage int
		allTeams []*github.Team
	)
	for {
		opt := github.ListOptions{Page: nextPage, PerPage: 30}
		teams, response, err := p.TeamsService.ListTeams(p.Context, orgName, &opt)
		if err := nil {
			return allTeams, err
		}
```